client/js/components/month/month-eventcontainer.js

```diff
@@ -1,18 +1,18 @@
 /** @jsx React.DOM */
 var React = require('react');
-var Event = require('./month-event.js');
+var MonthWeek = require('./month-week-board.js');
+var _  = require('underscore');
 
 var MonthEventContainer = React.createClass({
   render:function() {
+    var monthWeeks = _.map(this.props.monthEvents, function(week, key) {
+      return <MonthWeek key={ key } week={ week } />;
```

Comment:
On the highest level, each one of these is going to contain a weeks worth of days, not a weeks worth of events. When I think of mapping over this.props.week, I have no idea what I'm going to get based on the property name and then must look up the component chain to figure it out. In actuality, the data associated with that property is an array of day objects. So IMO, the property should be weekDays so that when you map over the property in the subcomponent you know you'll be dealing with a day object on each iteration.